src/components/ha-items-display-editor.ts

```diff
@@ -50,6 +51,9 @@ export class HaItemDisplayEditor extends LitElement {
   @property({ type: Boolean, attribute: "show-navigation-button" })
   public showNavigationButton = false;
 
+  @property({ type: Boolean, attribute: "disable-visible-sort" })
```

Comment:
It doesnt disable the sorting (as in, the user can still sort them) but we dont sort them...

Would `dont-sort-visible` be a better name?